sklearn/metrics/classification.py

```diff
@@ -2038,3 +2038,148 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
     y_true = np.array(y_true == pos_label, int)
     y_true = _check_binary_probabilistic_predictions(y_true, y_prob)
     return np.average((y_true - y_prob) ** 2, weights=sample_weight)
+
+
+def false_positive(y_true, y_pred, labels=None, sample_weight=None):
+    """Compute the false positive rate.
+
+    False positive rate indicate the
+    number of sample which are classified as
+    negative but they are positive.In
+    other word we can say that it is the
+    chances of occuring a TYPE-1 error.
+
+    Parameters
+    ----------
+    y_true : array, shape = [n_samples]
+        Ground truth (correct) target values.
+
+    y_pred : array, shape = [n_samples]
+        Estimated targets as returned by a classifier.
+
+    labels : array, shape = [n_classes], optional
+        List of labels to index the matrix. This may be used to reorder
+        or select a subset of labels.
+        If none is given, those that appear at least once
+        in ``y_true`` or ``y_pred`` are used in sorted order.
+
+    sample_weight : array-like of shape = [n_samples], optional
+        Sample weights.
+
+    Returns
+    -------
+    score = The number of times type-1 error has occurred.
+    """
+
+    # Checking the type of class which they belongs to
+    type_true = type_of_target(y_true)
+    type_pred = type_of_target(y_pred)
+
+    y_type = set([type_true, type_pred])
+    if len(y_type) == 1:
+        if y_type == set(["binary"]):
+            CM = confusion_matrix(y_true, y_pred, labels=labels,
+                                  sample_weight=sample_weight)
+            return CM[0][1]
+        else:
+            raise ValueError("this metric is supported for only binary class")
```

Comment:
Would be great if the Raise error message start with uppercase: `This metric ... ` this is valid for the whole raise message.

Reply:
@eamanu Can you tell me why my one check is failing

Reply:
I don't know Covecod, but I think that you should create testscases for the new features